### PR TITLE
[BOYSCOUT] Write some missing ADRs regarding recent work

### PR DESCRIPTION
**Pull request by chibenwa**
I did got some time in the plane...

### CODE REVIEW

File: src/adr/0060-adopt-bounded-elastic.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Status
+
+Accepted (lazy consensus).
+
+Implemented. 
+
+## Context
+
+See description of blocking vs reactive programming odel described in [ADR-57](0057-reactive-imap.md).
```
Comment (Arsnael) [inline src/adr/0060-adopt-bounded-elastic.md:13]:
```suggestion
See description of blocking vs reactive programming model described in [ADR-57](0057-reactive-imap.md).
```

File: src/adr/0060-adopt-bounded-elastic.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Sometime Reactive code needs to execute blocking tasks.
```
Comment (Arsnael) [inline src/adr/0060-adopt-bounded-elastic.md:15]:
```suggestion
Sometimes reactive code needs to execute blocking tasks. 
```

File: src/adr/0060-adopt-bounded-elastic.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ end up calling reactive data access code. This results in nested blocking calls. Nested blocking
+ calls, when using the same scheduler with a bounded thread cap, can result in a dead lock.
+
+To prevent such a dead-lock code executed on bounded-elastic should not depend on scheduling nested
+blocking call on bounded-elastic scheduler for its completion. We can thus avoid such a situation by 
+using a scheduler dedicated by nested blocking calls.
+
+## Alternatives
+
+Alternatives to the "blocking call wrapper" scheduler describes above includes a full reactive
```
Comment (Arsnael) [inline src/adr/0060-adopt-bounded-elastic.md:50]:
```suggestion
Alternatives to the "blocking call wrapper" scheduler described above includes a full reactive 
```

**Review by vttranlina — APPROVED** [review]

File: src/adr/0062-oidc-token-introspection.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Identity Provider server through the mean of a JWKS endpoint. Yet no effort is made to
+see if the access token in question was revoked or not, which can pause a security threat.
+
+OIDC ecosystem can support the following mechanisms to determine if an access token had been 
+revoked:
+
+ - Use of an introspection endpoint: the application asks the OIDC server to validate the token
+ through an HTTP call. This result in load on the identity provider, which becomes central to the
+ authentication process. This can be assimilated to a 'synchronous' mode.
+ - use of back-channel upon token revocation. The OIDC provider is then responsible to call an
```
Comment (quantranhong1999) [inline src/adr/0062-oidc-token-introspection.md:28]:
```suggestion
 - Use of back-channel upon token revocation. The OIDC provider is then responsible to call an 
```

**Review by quantranhong1999 — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by mbaechler** [comment]
:tada:   Thanks for these ADRs, it's all good stuff and it allows me to be up to date with what happen in james.
Good work folks!

**Comment by chibenwa** [comment]
Thanks @mbaechler and sorry for lacking the discipline to do it afterward...

**Review by jeantil — APPROVED** [review]

**Pull request merged**